### ISSUE TITLE
Hello, would you be able to do a build with this file? "meson.build"

### ISSUE THREAD

**Issue by NenelFX** [styler00dollar/VapourSynth-RIFE-ncnn-Vulkan#38]
Hello, would you be able to do a build with this file?
[meson.build.txt](https://github.com/styler00dollar/VapourSynth-RIFE-ncnn-Vulkan/files/14852296/meson.build.txt)
[meson.build.2.txt](https://github.com/styler00dollar/VapourSynth-RIFE-ncnn-Vulkan/files/14852422/meson.build.2.txt)

_Originally posted by @NenelFX in https://github.com/styler00dollar/VapourSynth-RIFE-ncnn-Vulkan/issues/14#issuecomment-2034519533_

**Comment by styler00dollar** [comment]
I don't understand why I should do that. The current meson.build does compile.

**Comment by NenelFX** [comment]
and why I don't know how to make one
"
Compilation
git submodule update --init --recursive --depth 1
meson build
ninja -C build
ninja -C build install
"
I don't even know which programs to use
and about the files
meson.build.txt
I got the version of "https://github.com/HomeOfVapourSynthEvolution/VapourSynth-RIFE-ncnn-Vulkan" using the "internet archive" website since it is no longer available on github
https://web.archive.org/web/20221022183912/https://github.com/HomeOfVapourSynthEvolution/VapourSynth-RIFE-ncnn-Vulkan

and because I wanted to test the other new Rife models since I had only been using Rifev4.6 for 2 years and recently tested the
  "https://github.com/TNTwise/rife-ncnn-vulkan"
and all rife models work on my pc.
But I would really like a version for the VapourSynth because I have a script that I use a lot on it.

I've already tried to make a Compilation but I don't know how to do it.

and the meson.build script from "HomeOfVapourSynthEvolution/VapourSynth-RIFE-ncnn-Vulkan"
It's very similar to your meson.build so if it were possible I wanted you to make a Compilation.

But I still don't understand why your version doesn't run on my PC.
And my last option is to buy a new PC in a few years.
